### DISCUSSION TITLE
Environment Import/Export or Selective Project Export

### DISCUSSION THREAD

**Discussion by andreassiegel** [Kong/insomnia#6203] [Ideas]
In Insomnia, it is possible to selectively export collections but environments cannot be imported and exported the same way. The only way is to export/import the entire Insomnia project which might be too much.

It would be nice to be able to handle export and import environments just like collections.

Alternatively, it would be great if it was possible to select what to include in a project export.

For instance, we have the strict requirement to work "offline", i.e., withouth any data being synced to the cloud, therefore, we are working with different collections in the "Insomia" default project/organization. These collections relate to different actual products of the same and we don't always want to export everything together.

**Comment by jandreaucodes** [comment]
Adding a +1 to this, as I'd find it incredibly helpful to be able to export a single Environment.

My use case is that I'd set up an Environment to do development work for a client and then want to share just that Environment and its variables with the client once development work is done, similar to how it'd be done in Postman.
